src/Widgets/CalendarWidget.php

```diff
@@ -59,8 +63,15 @@ class CalendarWidget extends Widget implements HasActions, HasForms
 
     protected int | string | array $columnSpan = 'full';
 
+    public array $currentView;
```

Comment:
Can you please remove the $currentView property and getView() method. Every event receives a `view` object already, so you are able to check it anyway.

Reply:
Hi, is there also a way to request the current date range when you haven't touched the event yet?

Reply:
I don't think so. Maybe we can add support for the `viewDidMount` event then and let people implement their own logic. What do you think?

Reply:
As a proof of concept i have added the viewDidMount logic and it does indeed do the trick. hope you like the implementation. Strange thing though.. according to the docs i should expect a view property but instead I receive the view as a object

Reply:
I refactored it a little so it works the same like the other events. You can enable it using the `viewDidMountEnabled` property and by overriding the onViewDidMount` method. There you can store the view or run any custom logic you want.